src/features/events/store.ts

```diff
@@ -93,6 +128,68 @@ const eventsSlice = createSlice({
       state.eventList = remoteList(action.payload);
       state.eventList.loaded = new Date().toISOString();
     },
+    filterTextAdded: (
```

Comment:
Instead of `filterTextAdded`, lets call this `filterTextUpdated` for consistency.

Reply:
Yeah I agree. I actually dreamt about this name last night. In my dream, I thought like "the name should be `filterTextUpdated` not `Added`. Cause it is not an array that user would add something in there."